### ISSUE TITLE
Theme Picker closing application

### ISSUE THREAD

**Issue by 0lmagalhaes0** [kivymd/KivyMD#661]
### Description of the Bug

When clicking on the button to open the dialog box from Theme Picker, the application closes itself with exit code 1.

### Code and Logs

```python
from kivy.lang import Builder

from kivymd.app import MDApp
from kivymd.uix.picker import MDThemePicker

KV = '''
FloatLayout:

    MDRaisedButton:
        text: "Open theme picker"
        pos_hint: {'center_x': .5, 'center_y': .5}
        on_release: app.show_theme_picker()
'''

class Test(MDApp):
    def build(self):
        return Builder.load_string(KV)

    def show_theme_picker(self):
        theme_dialog = MDThemePicker()
        theme_dialog.open()

Test().run()
```
Log:
```
[INFO   ] [Base        ] Start application main loop
[INFO   ] [Base        ] Leaving application in progress...
 Traceback (most recent call last):
   File "kivy\properties.pyx", line 2023, in kivy.properties.ColorProperty.convert
   File "kivy\properties.pyx", line 2041, in kivy.properties.ColorProperty.parse_list
 ValueError: Invalid value for color (got [])
 
 The above exception was the direct cause of the following exception:
 
 Traceback (most recent call last):
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\lang\builder.py", line 701, in _apply_rule
     setattr(widget_set, key, value)
   File "kivy\weakproxy.pyx", line 35, in kivy.weakproxy.WeakProxy.__setattr__
   File "kivy\properties.pyx", line 498, in kivy.properties.Property.__set__
   File "kivy\properties.pyx", line 527, in kivy.properties.Property.set
   File "kivy\properties.pyx", line 2025, in kivy.properties.ColorProperty.convert
 ValueError: MDTabsLabel.color has an invalid format (got [])
 
 During handling of the above exception, another exception occurred:
 
 Traceback (most recent call last):
   File "F:/Projects/Python/themes/themes.py", line 25, in <module>
     Test().run()
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\app.py", line 950, in run
     runTouchApp()
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\base.py", line 582, in runTouchApp
     EventLoop.mainloop()
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\base.py", line 347, in mainloop
     self.idle()
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\base.py", line 391, in idle
     self.dispatch_input()
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\base.py", line 342, in dispatch_input
     post_dispatch_input(*pop(0))
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\base.py", line 308, in post_dispatch_input
     wid.dispatch('on_touch_up', me)
   File "kivy\_event.pyx", line 709, in kivy._event.EventDispatcher.dispatch
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivymd\uix\behaviors\ripplebehavior.py", line 245, in on_touch_up
     return super().on_touch_up(touch)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivymd\uix\button.py", line 1083, in on_touch_up
     return super().on_touch_up(touch)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivymd\uix\button.py", line 969, in on_touch_up
     return super().on_touch_up(touch)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\uix\behaviors\button.py", line 179, in on_touch_up
     self.dispatch('on_release')
   File "kivy\_event.pyx", line 705, in kivy._event.EventDispatcher.dispatch
   File "kivy\_event.pyx", line 1248, in kivy._event.EventObservers.dispatch
   File "kivy\_event.pyx", line 1132, in kivy._event.EventObservers._dispatch
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\lang\builder.py", line 57, in custom_callback
     exec(__kvlang__.co_value, idmap)
   File "<string>", line 7, in <module>
   File "F:/Projects/Python/themes/themes.py", line 21, in show_theme_picker
     theme_dialog = MDThemePicker()
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivymd\theming.py", line 907, in __init__
     super().__init__(**kwargs)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\uix\floatlayout.py", line 65, in __init__
     super(FloatLayout, self).__init__(**kwargs)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\uix\modalview.py", line 187, in __init__
     super(ModalView, self).__init__(**kwargs)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\uix\anchorlayout.py", line 68, in __init__
     super(AnchorLayout, self).__init__(**kwargs)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\uix\layout.py", line 76, in __init__
     super(Layout, self).__init__(**kwargs)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivymd\uix\behaviors\backgroundcolorbehavior.py", line 150, in __init__
     super().__init__(**kwargs)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\uix\widget.py", line 361, in __init__
     rule_children=rule_children)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\uix\widget.py", line 465, in apply_class_lang_rules
     rule_children=rule_children)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\lang\builder.py", line 543, in apply
     rule_children=rule_children)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\lang\builder.py", line 664, in _apply_rule
     child, crule, rootrule, rule_children=rule_children)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\lang\builder.py", line 659, in _apply_rule
     child = cls(__no_builder=True)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\uix\boxlayout.py", line 145, in __init__
     super(BoxLayout, self).__init__(**kwargs)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\uix\layout.py", line 76, in __init__
     super(Layout, self).__init__(**kwargs)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivymd\uix\tab.py", line 374, in __init__
     self.tab_label = MDTabsLabel(tab=self)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivymd\uix\tab.py", line 328, in __init__
     super().__init__(**kwargs)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\uix\behaviors\togglebutton.py", line 84, in __init__
     super(ToggleButtonBehavior, self).__init__(**kwargs)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\uix\behaviors\button.py", line 121, in __init__
     super(ButtonBehavior, self).__init__(**kwargs)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\uix\label.py", line 318, in __init__
     super(Label, self).__init__(**kwargs)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\uix\widget.py", line 361, in __init__
     rule_children=rule_children)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\uix\widget.py", line 465, in apply_class_lang_rules
     rule_children=rule_children)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\lang\builder.py", line 543, in apply
     rule_children=rule_children)
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\lang\builder.py", line 712, in _apply_rule
     e), cause=tb)
 kivy.lang.builder.BuilderException: Parser: File "<inline>", line 19:
 ...
      17:        )
      18:    color:
 >>   19:        self.text_color_active if self.state == 'down'         else self.text_color_normal
      20:    on_x: self._trigger_update_tab_indicator()
      21:    on_width: self._trigger_update_tab_indicator()
 ...
 ValueError: MDTabsLabel.color has an invalid format (got [])
   File "F:\Anaconda\envs\AppMD\lib\site-packages\kivy\lang\builder.py", line 701, in _apply_rule
     setattr(widget_set, key, value)
   File "kivy\weakproxy.pyx", line 35, in kivy.weakproxy.WeakProxy.__setattr__
   File "kivy\properties.pyx", line 498, in kivy.properties.Property.__set__
   File "kivy\properties.pyx", line 527, in kivy.properties.Property.set
   File "kivy\properties.pyx", line 2025, in kivy.properties.ColorProperty.convert
 

Process finished with exit code 1
```

### Screenshots

![image](https://user-images.githubusercontent.com/61873279/98798036-b385d780-240d-11eb-90ce-37da392f41a7.png)

### Versions

* OS: Windows 10
* Python: 3.6
* Kivy: 1.11.1
* KivyMD: 0.104.1

**Comment by kulothunganug** [comment]
@0lmagalhaes0 this error had fixed in `0.104.2.dev0` (master version)
use `pip install https://github.com/kivymd/KivyMD/archive/master.zip` to install latest version of `KivyMD`

**Comment by 0lmagalhaes0** [comment]
Cool, sorry to bother.
Thanks!
